javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/extension/instrumentation/InstrumentationModule.java

```diff
@@ -159,4 +160,15 @@ public ElementMatcher.Junction<ClassLoader> classLoaderMatcher() {
   public List<String> getAdditionalHelperClassNames() {
     return Collections.emptyList();
   }
+
+  /**
+   * Only functional for Modules where {@link #isIndyModule()} returns {@code true}.
+   *
+   * <p>Normally, helper and advice classes are loaded in a child classloader of the instrumented
+   * classloader. This method allows to inject classes directly into the instrumented classloader
+   * instead.
+   *
+   * @param injector the builder for injecting classes
+   */
+  public void injectClasses(ClassInjector injector) {}
```

Comment:
WDYT instead introducing an interface, something like `ExtendedInstrumentationModule` and placing this method there. Eventually when we are happy with the api we can move it back to `nstrumentationModule`.

Reply:
Sounds good!